packages/tools/devtools/devtools-example/.eslintrc.js

```diff
@@ -31,6 +32,10 @@ module.exports = {
 			rules: {
 				"import/no-nodejs-modules": "off",
 				"unicorn/prefer-module": "off",
+				"unicorn/prefer-query-selector": "off",
+				"@typescript-eslint/no-unsafe-argument": "off",
```

Comment:
Do we need to disable these typescript-eslint rules? These would be good for us to abide by in our tests too.

Reply:
Removed